actions.toml

```diff
@@ -15,3 +15,41 @@ command = "/bin/bash scripts/actions/reset_default_config \"pihole-FTL.conf\""
 # accepted_return_codes = [0, 1, 2, 3]  # optional
 accepted_return_codes = [0]
 description = "Reset the config file pihole-FTL.conf."
+
+[reset_default_dnsmasq]
+name = "Reset the config file and restore a default one."
+command = "/bin/bash scripts/actions/reset_default_config \"01-pihole.conf\""
+# user = "root"  # optional
+# cwd = "/" # optional
+# accepted_return_codes = [0, 1, 2, 3]  # optional
+accepted_return_codes = [0]
+description = "Reset the config file dnsmasq.d/01-pihole.conf."
+
+
+[reset_default_nginx]
+name = "Reset the nginx config for this app."
+command = "/bin/bash scripts/actions/reset_default_system nginx"
+# user = "root"  # optional
+# cwd = "/" # optional
+# accepted_return_codes = [0, 1, 2, 3]  # optional
+accepted_return_codes = [0]
+description = "Reset the nginx config for this app."
```

Comment:
So hmpf we were discussing actions with ljf this afternoon and it's really madness to define action for this in like ~every app that has an actions.toml ... If there's a need for a mecanism to regen nginx/php/systemd services etc, let's define a global mecanism in the core. But copypasting this in every app is madness and unmanageable on the long run...

Reply:
Yeah, I don't know what to do with that... I agree with you, but since it's already done, I think we can leave it as is.